Tools/MiniBrowser/gtk/main.c

```diff
@@ -76,6 +76,7 @@ static gboolean enableITP;
 static gboolean exitAfterLoad;
 static gboolean webProcessCrashed;
 static gboolean printVersion;
+static char* configFile;
```

Comment:
So this file is a little confusing due to the mix of different code styles. With regards to the asterisks, your pull request uses a mix of our C++ style (where asterisk hangs on the type) vs. our C style (where asterisk hangs on the variable name, like GNOME style). The rest of this file puts it on the variable name.

```suggestion
static char *configFile;
```

Reply:
Oops, I thought the style-checker was reporting these :)